signer/src/error.rs

```diff
@@ -26,6 +26,10 @@ pub enum Error {
     #[error("sweep transaction not found: {0}")]
     MissingSweepTransaction(bitcoin::Txid),
 
+    /// Indicates that a deposit request with the specified txid and vout could not be found.
+    #[error("deposit request not found: {0}, {1}")]
+    MissingDepositRequest(bitcoin::Txid, u32),
```

Comment:
Use the `OutPoint` type here, it's simpler.

Reply:
done in https://github.com/stacks-network/sbtc/pull/956/commits/c803d9d18bc48fc3aae449ca3c59233a26f44228

Reply:
changed in https://github.com/stacks-network/sbtc/pull/1005/commits/23b091cb0204cde75f0f5f0d96662a983b6febe0